src/main/java/de/dhbw/karlsruhe/students/mailflow/external/infrastructure/email/GSONConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package de.dhbw.karlsruhe.students.mailflow.external.infrastructure.email;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonToken;
+import com.google.gson.stream.JsonWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+
+public class GSONConfiguration {
+  private GSONConfiguration() {}
```

Comment:
Do we need the implementation of the standard constructor?

Reply:
Yes this constructor is now private. That means you will not be able to accidentally create an instance of this GSONConfiguration. This is helpful because the GSONConfiguration provides only static methods